### PR TITLE
Excel: stop freeze when detecting overflow/cropped text

### LINKED ISSUE #6216: In excel, a certain document causes freezes
I will send the document to @MichaelDCurran Privately.
1. Open the excel document.
2. Navigate down to row 9. Other rows in the 40's freeze NVDA as well.
3. Expected behavior: Contents are read. 
Behavior: NVDA freezes.
This is being sent privately because I don't want the document in the public since I didn't create it, and can't figure out how to reproduce.

### LINKED ISSUE #6472: Excel: Text which isn't too wide, announced as overflowing / cropped when using high DPI
When the DPI is changed in Windows (usually via the "Make text and other items larger or smaller" setting), it makes everything bigger or smaller - text, icons, the size of cells in Excel.  The text "abcdefghi" fits neatly in a default sized cell at the default Calibri 11pt font.  This is the same regardless of the DPI as while a higher DPI makes the text bigger, it also makes the cells in Excel bigger, so the same text still fits.  There is a small discrepancy - I've found that you can fit a "j" on the end in some conditions.

When the DPI is set above 100%, NVDA seems to behave as if the text has got bigger but the size of the cell hasn't (just a guess).  At 200%, I can about half fill a cell before NVDA announces it as overflowing (or cropped if there is text in the next cell).  I tested this on Windows 10 with Excel 2016, and Windows 7 with Excel 2007.

To reproduce:
- Set "Make text and other items larger or smaller" to anything above 100% - the higher the value, the more obvious the effect.
- Sign out / back in.
- Open Excel
- Type text in a cell.  The text "abcdefghi" fits in a cell at the default Calibri 11 under every condition I tried.
- Navigate away from the cell and back (tab then shift+tab).  NVDA reads the text and reports "overflowing".

### PR DESCRIPTION
**Pull request by michaelDCurran**
Fixes #6216, Fixes #6472

### CODE REVIEW

**Comment by michaelDCurran** [comment]
@feerrenrut: A review please. I confirmed this fixes Derek's testcase mentioned on #6216, and @qchristensen confirms this fixes #6472

File: source/NVDAObjects/window/excel.py [hunk 1]
```diff
@@ -1219,14 +1217,13 @@ class structText(ctypes.Structure):
 		#Release & Delete the device context
 		ctypes.windll.gdi32.DeleteDC(tempDC)
 		#Retrieve the text width
-		textWidth = StructText.width+5
-		cellWidth  = self.excelCellObject.ColumnWidth * xlCellWidthUnitToPixels	#Conversion factor to convert the cellwidth to pixels
-		return (cellWidth,textWidth)
+		textWidth = StructText.width
+		return textWidth #ctypes.windll.kernel32.MulDiv(textWidth, 96, dpi)
```
Comment (feerrenrut) [inline source/NVDAObjects/window/excel.py]: I dont understand the comment on the end of this. Was it left there by mistake?

**Review by feerrenrut — COMMENTED** [review]
Looks good

File: source/NVDAObjects/window/excel.py (hunk 1 above)
Comment (michaelDCurran) [inline source/NVDAObjects/window/excel.py]: Yep, left there by mistake, I shall remove.

**Pull request merged**